frontend/src/app/search/loading.tsx

```diff
@@ -1,27 +1,13 @@
 import React from "react";
+import Spinner from "../../components/Spinner";
 
 export default function Loading() {
-  const listStyle: React.CSSProperties = {
-    display: "flex",
-    flexDirection: "column",
-    alignItems: "center",
-    height: "50vh",
-    listStyleType: "none",
-  };
-
-  const skeletonStyle = {
-    backgroundColor: "#eee",
-    borderRadius: "4px",
-    height: "20px",
-    margin: "10px 0",
-    width: "50%",
-  };
-
   return (
-    <ul style={listStyle}>
-      {Array.from({ length: 10 }).map((_, index) => (
-        <li key={index} style={skeletonStyle} />
-      ))}
-    </ul>
+    <div className="display-flex flex-align-center flex-justify-center margin-bottom-15 margin-top-15">
+      <Spinner />
+      <span className="font-body-2xl text-bold margin-left-2">
+        Loading results...
```

Comment:
This should be run through translation, and could be after #1657 , which we will hopefully pull in today. We could wait or at least add a TODO in the meantime and update in #1657 .

Reply:
Thanks - added a todo and ticket for it